src/models/ComponentStyle.js

```diff
@@ -12,10 +39,15 @@ export default (nameGenerator: NameGenerator, flatten: Flattener, stringifyRules
   class ComponentStyle {
     rules: RuleSet
     componentId: string
+    isStaticRules: boolean
+    staticGeneratedClassName: ?string = null
```

Comment:
We generally prefer `undefined` in the codebase :) Also this way it won't need to be initialised